.rubocop_todo.yml

```diff
@@ -1,34 +1,47 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2015-10-20 14:00:00 -0700 using RuboCop version 0.34.2.
+# on 2015-10-27 17:33:47 -0700 using RuboCop version 0.34.2.
```

Comment:
Before we merge, I'd like to re-run this, and hopefully see most of the introduced offenses go away. Ideally, our new code won't introduce new violations (except as needed to avoid introducing radically different code styles) and we'll leave the codebase better than we found it.